src/app/projects/states/portfolio/directives/portfolio-grade-select-step/portfolio-grade-select-step.tpl.html

```diff
@@ -33,10 +33,7 @@ <h4>Grade Application</h4>
             ng-model="project.submitted_grade"
             btn-radio="{{$index}}"
           >
-            <grade-icon
-              grade="grade"
-              class="text-{{project.submitted_grade == grades.indexOf(grade) ? 'success' : 'muted'}}"
-            ></grade-icon>
```

Comment:
Are we keeping this styling somehow? (not sure that we need to)

Reply:
Unless I'm mistaken, it didn't appear that these styles were being used. Comparing the new component with the old, I believe they both look identical.